nixos/modules/rename.nix

```diff
@@ -250,6 +250,9 @@ with lib;
     (mkRenamedOptionModule [ "programs" "info" "enable" ] [ "documentation" "info" "enable" ])
     (mkRenamedOptionModule [ "programs" "man"  "enable" ] [ "documentation" "man"  "enable" ])
 
+    # PAM
+    (mkRenamedOptionModule [ "security" "pam" "enableU2F" ] [ "security" "pam" "u2f" "enable" ])
+
```

Comment:
Isn't this missing the u2fAuth -> u2f.enable change?

Reply:
That's defined on every (freeform) service. I don't believe the rename module supports a usecase where something like `security.pam.services.<name?>.u2fAuth` is renamed.

Reply:
Ah right, it won't work here probably, but maybe it'll work by adding an `imports = [(mkRenamed ...)]` at line 12 in pam.nix